std/uuid.d

```diff
@@ -1694,7 +1694,8 @@ public class UUIDParsingException : Exception
     ///The position in the input string where the error occurred.
     size_t position;
 
-    private this(string input, size_t pos, Reason why = Reason.unknown, string msg = "",
+    ///
+    this(string input, size_t pos, Reason why = Reason.unknown, string msg = "",
```

Comment:
It doesn't look like user code is intended to throw this exception. I think that instead of making the constructor public, the unittest/example needs to be converted to a private unittest, and a new unittest/example take its place, i.e. one which invokes a public std.uuit method with some invalid data that causes an instance of this exception type to be thrown.